modules/stochastic_tools/include/utils/PolynomialQuadrature.h

```diff
@@ -87,28 +87,29 @@ class Legendre : public Polynomial
 class Hermite : public Polynomial
 {
 public:
+  Hermite(const Real mu, const Real sig);
   Hermite(const NormalDistribution * dist);
```

Comment:
@zachmprince I am thinking that we don't need these constructors. The make_polynomial function can take a distribution and then just call the constructor I created above. What to you think?

Reply:
That could be a good idea. Do you think that would look cleaner? We would still need to dynamic cast and gather input parameters.

Reply:
Yes, the makePolynomial would be about the same. But, the Polynomial objects would be cleaner. They are simply selected based on the distribution but do not have a direct dependence. I will add this to the PR.